src/main/java/ru/yandex/practicum/filmorate/FilmorateApplication.java

```diff
@@ -2,27 +2,19 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
-// Почему прописал @NotNull для даты релиза? Перечитай ТЗ - валидация по дате релиза присутствует в методе валидации
-// а проверка на null просто как дополнительная проверка? т.к. если туда попадет null-будет npe
+// Не поправил
+// Чтобы не отлавливать руками такие вещи (плюс всякие отступы и пробелы), используй комбинацию Ctrl+Alt+L- я не знаю почему,
+// у меня там один таб
 
-//А почему для каких-то полей ты прописываешь final, а для каких-то нет? - для name не прописываю final
-// т.к. его нужно изменять если поле пустое на значение login*a
+//Если name=null, что выдаст выражение user.getName().isBlank()? - будет npe, но там же для этого есть проверка на null
```

Comment:
https://ru.stackoverflow.com/questions/712136/%D0%92-%D1%87%D1%91%D0%BC-%D0%BE%D1%82%D0%BB%D0%B8%D1%87%D0%B8%D0%B5-isblank-vs-isempty